mobify/astro-scaffold
# Add linting to the scaffold

Add JS linting to the Astro scaffold

Reviewers: @jansepar @kschmidtdev @jvoll @jvaill @MikeKlemarewski @jeremywiebe @lizcross 
JIRA: **https://mobify.atlassian.net/browse/HYB-481**
Linked PRs: **https://github.com/mobify/astro/pull/291**
## Changes
- Refactor Grunt tasks
- Add linting task
- Fix all existing JS linting errors
- Make linting a part of the build process
## How to test-drive this PR
- Checkout code
- Make a linting error
- Build and make sure error is caught and fails the build
## TODOS:
- [x] Use Astro `develop` branch once https://github.com/mobify/astro/pull/291 is merged
